src/component/Maps/maps.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import "./maps.scss";
+import GoogleMapReact from "google-map-react";
+
+
+import { Icon } from "@iconify/react";
+import locationIcon from "@iconify/icons-mdi/map-marker";
+import Polyline from "google-map-react";
+const google_maps_key = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
+
+const Location1 = ({ text1 }) => (
+  <div className="LocationPin">
+    <Icon icon={locationIcon} className="LocationPin__icon" />
+  </div>
+);
+const Location2 = ({ text2 }) => (
+  <div className="LocationPin">
+    <Icon icon={locationIcon} className="LocationPin__icon" />
+  </div>
+);
```

Comment:
I'm wondering if there is a way to create less static code. These two components seem duplicates of one another, even though they hold different values. We can try and create an object and use the `Object.map()` function to try and get rid of some of the duplicate code here. Check out the way we did the `SelectMenu` components. 

https://github.com/CSE115A/rideshare-project/blob/dab65495e206993004daca2192060d22224801bb/src/component/AddressForm/AddressForm.jsx#L29-L40

This mapping is a result of creating the following array object 
https://github.com/CSE115A/rideshare-project/blob/dab65495e206993004daca2192060d22224801bb/src/component/AddressForm/AddressForm.jsx#L11-L26